packages/metascraper-author/index.js

```diff
@@ -4,6 +4,7 @@ const { getValue, isUrl, titleize } = require('@metascraper/helpers')
 const { isString } = require('lodash')
 
 const REGEX_STRICT = /^\S+\s+\S+/
+const REGEX_DATE = /\d{2,}/g
```

Comment:
Hmmm not sure if global flag is necessary, what do you think?

Reply:
Your right! It's not needed.